app/Http/Controllers/Apis/Protected/Summit/Factories/SponsorValidationRulesFactory.php

```diff
@@ -40,10 +40,21 @@ public static function buildForAdd(array $payload = []): array
             'header_image_alt_text' => 'sometimes|string|max:255',
             'header_image_mobile_alt_text' => 'sometimes|string|max:255',
             'carousel_advertise_image_alt_text' => 'sometimes|string|max:255',
-            'sponsorships' => 'sometimes|sponsorship_dto_array',
+            'sponsorships' => 'sometimes|int_array',
```

Comment:
@romanetar this is not correct
oringal param was
'sponsorship_id' => 'required|integer',